miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniApp.kt

```diff
@@ -30,23 +30,6 @@ abstract class MiniApp internal constructor() {
     @Throws(MiniAppSdkException::class)
     abstract suspend fun listMiniApp(): List<MiniAppInfo>
 
-    /**
-     * Creates a mini app.
-     * The mini app is downloaded, saved and provides a [MiniAppDisplay] when successful.
-     * @param appId mini app id.
-     * @param miniAppMessageBridge the interface for communicating between host app & mini app
-     * @throws [MiniAppNotFoundException] when the specified mini app ID does not exist on the server
-     * @throws [MiniAppHasNoPublishedVersionException] when the specified mini app ID exists on the
-     * server but has no published versions
-     * @throws [MiniAppSdkException] when there is any other issue during fetching,
-     * downloading or creating the view.
-     */
-    @Throws(MiniAppNotFoundException::class, MiniAppHasNoPublishedVersionException::class, MiniAppSdkException::class)
-    abstract suspend fun create(
-        appId: String,
-        miniAppMessageBridge: MiniAppMessageBridge
-    ): MiniAppDisplay
-
     /**
      * Same as [create(String, MiniAppMessageBridge)].
```

Comment:
This line now looks redundant because `create(String, MiniAppMessageBridge)` doesn't exist.